mvlearn/cluster/mv_kmeans.py

```diff
@@ -154,7 +164,8 @@ class MultiviewKMeans(BaseKMeans):
     '''
 
     def __init__(self, n_clusters=2, random_state=None, init='k-means++',
-                 patience=5, max_iter=None, n_init=5):
```

Comment:
can you reorder the parameters, either here or in the docstrings, so they're the same order?

Reply:
Will do